ci_tests/test_pygoat_findings.py

```diff
@@ -6,7 +6,9 @@
     "pixee:python/add-requests-timeouts",
     "pixee:python/secure-random",
     "pixee:python/sandbox-process-creation",
+    "pixee:python/subprocess-shell-false",
```

Comment:
This one is a result of adding `--dry-run` (which I was using locally while testing).